### PR TITLE
Popover & Tooltip doc enhancements

### PR DESCRIPTION
**Pull request by driskull**
- Mark referenceElement as required for Tooltip and Popover.
- Add basic usage examples for Tooltip and Popover.
- This PR serves as an example of how we can add code snippets or usage information that will be auto generated into the readme.

cc @kat10140 for reference.

### CODE REVIEW

File: src/components/calcite-popover/readme.md [hunk 1]
```diff
@@ -3,22 +3,40 @@
 <!-- Auto Generated Below -->
 
 
+## Usage
+
+### AddClickHandle
+
+```html
+<calcite-popover reference-element="popover-button" add-click-handle
```
Comment (paulcpederson) [inline src/components/calcite-popover/readme.md]: Is there a condition where we don't want a popover to open when the reference element is clicked? For my use cases the overwhelming majority will require this prop, but not sure what other teams needs are?

**Comment by paulcpederson** [comment]
LGTM, kind of a clunky prop name, but works for me

File: src/components/calcite-popover/readme.md (hunk 1 above)
Comment (driskull) [reply src/components/calcite-popover/readme.md]: Yeah, in the MapViewer there are some use cases where the reference element is only used for positioning. For example: popover is displayed alongside a panel.

**Comment by driskull** [comment]
> LGTM, kind of a clunky prop name, but works for me

Agreed. The prop was already there before this PR though. Since you guys are still in beta we can easily rename and break things :)

**Pull request merged**